src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -5,12 +5,23 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
-
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace System.Net.Http.WinHttpHandlerTests
 {
+    public class testHandler:WinHttpHandler
```

Comment:
@davidsh  SendAsync is a protected function, so I can only access to it by defining a class that inherits from WinHttpHandler and exposes that function. 

@CIPop  Ok, I'll separate that class in its own .cs.